### ISSUE TITLE
Broken semantic search scoring link

### ISSUE THREAD

**Issue by cjcenizal** [vectara/vectara-docs#133]
@pwoznic https://docs.vectara.com/docs/common-use-cases/semantic-search/semantic-search-scoring is a broken link. Unfortunately it's also a Google search result for ["vectara docs tuning"](https://www.google.com/search?q=vectara+docs+tuning).

Can we do an RCA to figure out how this link broken and if any other links were broken by the same root cause?

**Comment by pwoznic** [comment]
I think Google has not yet indexed the new URLs from the IA revamp. I'm wondering if we can just update the redirects to include old links.

**Comment by cjcenizal** [comment]
Was "https://docs.vectara.com/docs/common-use-cases/semantic-search/semantic-search-scoring" a working URL before the IA revamp?

> I'm wondering if we can just update the redirects to include old links.

👍 I think we should use the redirects to ensure all links that Google is sending people to, resolve to useful pages.

**Comment by cjcenizal** [comment]
@pwoznic Thanks for fixing this via https://github.com/vectara/vectara-docs/pull/135#event-10815883413! Do we know what caused the broken link? Was it due to a recent change or has this been a broken link for awhile?

**Comment by pwoznic** [comment]
> @pwoznic Thanks for fixing this via [#135 (comment)](https://github.com/vectara/vectara-docs/pull/135#event-10815883413)! Do we know what caused the broken link? Was it due to a recent change or has this been a broken link for a while?

It was due to the IA change and some topics were moved around and/or renamed. Google still referenced the old topic in its search results. I went through a G search testing out each topic and then updated several references with redirects once I got errors. Let me know if you find others. Also, I can update the comment on 135 if you want me to close this one (wanted to wait). :)
